modules/installation-user-infra-machines-pxe.adoc

```diff
@@ -139,6 +139,11 @@ file on your TFTP server. The `coreos.inst.image_url` parameter value is the
 location of the compressed metal RAW image on your HTTP server, and the
 `coreos.inst.ignition_url` parameter value is the location of the bootstrap
 Ignition config file on your HTTP server.
++
+[NOTE]
+====
+You can add use serial console by appending `console=ttyS0`.
```

Comment:
You can use serial console  (same for the other instances in this PR)

Reply:
Updated!